torch/distributed/fsdp/_optim_utils.py

```diff
@@ -84,7 +98,7 @@ class _OptimStateKey(NamedTuple):
     """
 
     unflat_param_names: Tuple[str, ...]
-    is_flat_param: bool
+    is_fsdp_wrapped: bool
```

Comment:
nit: Should we call this `is_fsdp_managed` (since no more wrapping in the composable path)?

For FSDP + Dynamo, we decided on using the word "managed" as well.